app/(main)/settings/team/components/agent-time-usage.tsx

```diff
@@ -0,0 +1,158 @@
+import { Alert, AlertDescription, AlertTitle } from "@/components/ui/alert";
+import { AlertTriangleIcon } from "lucide-react";
+
+export function AgentTimeUsageForFreePlan({
+	usedMinutes,
+}: { usedMinutes: number }) {
+	const includedMinutes = 60;
+	const warningMessage =
+		"Your free plan limit is almost reached. Upgrade to Pro for unlimited usage.";
+	const overLimitMessage =
+		"You've reached your free plan limit. Please upgrade to continue using the service.";
+	return (
+		<AgentTimeUsage
+			usedMinutes={usedMinutes}
+			includedMinutes={includedMinutes}
+			warningMessage={warningMessage}
+			overLimitMessage={overLimitMessage}
+		/>
+	);
+}
+
+export function AgentTimeUsageForProPlan({
+	usedMinutes,
+}: { usedMinutes: number }) {
+	const includedMinutes = 120;
+	const warningMessage =
+		"Getting close to your monthly limit. Usage beyond this limit will be billed additionally";
+	const overLimitMessage = "Additional minutes are subject to extra charges";
+
+	return (
+		<AgentTimeUsage
+			usedMinutes={usedMinutes}
+			includedMinutes={includedMinutes}
+			warningMessage={warningMessage}
+			overLimitMessage={overLimitMessage}
+		/>
+	);
+}
+
+type AgentTimeUsageProps = {
+	/** Minutes used */
+	usedMinutes: number;
+	/** Minutes included in the plan */
+	includedMinutes?: number;
+	/** Warning threshold ratio (decimal between 0-1) */
+	warningThreshold?: number;
+	/** Message displayed when approaching threshold */
+	warningMessage?: string;
+	/** Message displayed when exceeding limit */
+	overLimitMessage?: string;
+};
+
+function AgentTimeUsage(props: AgentTimeUsageProps) {
+	const {
+		usedMinutes,
+		includedMinutes = 120,
```

Comment:
The value `120` appears multiple times, so I think it would be better to define it as a constant, at least within this file.

Reply:
🛠️ https://github.com/giselles-ai/giselle/pull/200/commits/6fc7b1bd26dff1ec8ff378fdb4a39b5b163b2e54